### DISCUSSION TITLE
(Suggestion) Can we have a sidebar for password manager like bookmarks and history without opening in a new tab but customized and optimized as a sidebar panel?

### DISCUSSION THREAD

**Discussion by ghost** [Floorp-Projects/Floorp#1555] [Feature Request]
Can we have a sidebar for password manager like bookmarks and history without opening in a new tab but customized and optimized as a sidebar panel?

I added a shortcut but opens in a new tabs and it causes a lot of inconveniences. 

And a little bit of reduction in the size of the sidebar as it looks very big. 

![Screenshot 2024-12-26 100556](https://github.com/user-attachments/assets/78fb7cb4-821f-46bf-a0bd-7ee7f81d58ed)

**Comment by LiamillionSS** [comment]
Password manager extensions like bitwarden work in the sidebar, I know its not a solution to your question but there are alternatives
